cmd/controller/main.go

```diff
@@ -28,9 +30,20 @@ const (
 	ControllerLogKey = "controller"
 )
 
+var (
+	entrypointImage = flag.String("entrypoint-image", "override-with-entrypoint:latest",
+		"The container image containing our entrypoint binary.")
+	nopImage = flag.String("nop-image", "override-with-nop:latest",
+		"The container image used to kill sidecars")
+)
+
 func main() {
+	images := map[string]string{
```

Comment:
Maybe an `Images` struct with named fields? Having a `map` makes it seem like people should be able to add any image they want, and invites typos.